interceptors/unary_interceptors.go

```diff
@@ -141,7 +141,7 @@ func HystrixClientInterceptor() grpc.UnaryClientInterceptor {
 		newCtx, cancel := context.WithCancel(ctx)
 		defer cancel()
 		var err error
-		_ = hystrix.Do(options.cmdName, func() (e error) {
+		herr := hystrix.Do(options.cmdName, func() (e error) {
```

Comment:
could you also add unit tests to ensure we can get errors from hystrix itself https://github.com/carousell/Orion/blob/92b31a1e1157fd7355ee449a91c9307cb464392e/interceptors/interceptors_test.go#L36